--- rootfs.orig/etc/sudoers
+++ rootfs/etc/sudoers
@@ -19,8 +19,8 @@ Defaults	secure_path="/usr/local/sbin:/u
 # User privilege specification
 root	ALL=(ALL:ALL) ALL
 
-# Allow members of group sudo to execute any command
-%sudo	ALL=(ALL:ALL) ALL
+# Allow members of group sudo to execute any command without password
+%sudo	ALL=(ALL:ALL) NOPASSWD:ALL
 
 # See sudoers(5) for more information on "#include" directives:
 
