--- rootfs.orig/etc/dhcp/dhcpd.conf
+++ rootfs/etc/dhcp/dhcpd.conf
@@ -10,20 +10,22 @@
 ddns-update-style none;
 
 # option definitions common to all supported networks...
-option domain-name "example.org";
-option domain-name-servers ns1.example.org, ns2.example.org;
+option domain-name "barnacleos.local";
+option domain-name-servers 8.8.8.8, 8.8.4.4;
 
 default-lease-time 600;
 max-lease-time 7200;
 
 # If this DHCP server is the official DHCP server for the local
 # network, the authoritative directive should be uncommented.
-#authoritative;
+authoritative;
 
 # Use this to send dhcp log messages to a different log file (you also
 # have to hack syslog.conf to complete the redirection).
 log-facility local7;
 
+include "/etc/dhcp/dhcpd.conf.d/192.168.82.0.conf";
+
 # No service will be given on this subnet, but declaring it helps the 
 # DHCP server to understand the network topology.
 
